test/input-test.html

```diff
@@ -205,6 +205,11 @@
           });
         });
 
+        it(`should reflect to attribute when readonly property is set`, () => {
+          timePicker['readonly'] = true;
```

Comment:
Please follow we style we use elsewhere:

```suggestion
          timePicker.readonly = true;
```

Reply:
done!